.github/workflows/test_adaptersv2.yaml

```diff
@@ -148,17 +154,15 @@ jobs:
           echo "this is $svcip"
           config='{"contexts":{"local":{"endpoint":'$svcip',"token":"Default","platform":"kubernetes","adapters":[],"channel":"stable","version":"latest"}},"current-context":"local","tokens":[{"location":"auth.json","name":"Default"}]}'
           echo $config  > ~/.meshery/config.yaml
-      - name: Download patternfile
```

Comment:
This block failed. How does it work? from where it downloads patternfile?

Reply:
This downloads the patternfile which is uploaded before applying this job in the `SetPatternfileAction`